README.md

```diff
@@ -10,6 +10,14 @@ See https://uniresolver.io/ for a publicly hosted instance of a Universal Resolv
 
 ## Quick Start
 
+In order to access driver-did-trustbloc image you have to create personal token with read:packages and repo permissions ([personal token](https://help.github.com/en/github/authenticating-to-github/creating-a-personal-access-token-for-the-command-line)).
```

Comment:
Is this required for pulling the image from Github? Could you host it on DockerHub instead, to avoid the need for an access token?

Reply:
@peacekeeper thank you for the comment. We pushed the image to DockerHub and I removed this section from README.md